src/ProductBundle/Admin/ProductAdmin.php

```diff
@@ -169,7 +169,7 @@ public function getProduct()
     public function validate(ErrorElement $errorElement, $object): void
     {
         $errorElement
-            ->assertCallback(['validateOneMainCategory'])
+            ->assertCallback(['callback' => 'validateOneMainCategory'])
```

Comment:
Same here, are you sure this does not need to go to the stable branch too?

Reply:
In Symfony 2.8, if you didn't specify a `callback` or `methods` key, it added it automatically. This changed in 3.0 where the key is now required. So although this change will work on Symfony 2.8 as well, it is not required so I think it's fine to only target the dev branch